source/voice_control/s3_box_voice_assistant.markdown

```diff
@@ -28,9 +28,11 @@ Before you can use this device with Home Assistant, you need to install a bit of
 
 1. Make sure this page is opened in a Chromium-based browser on a **desktop**. The software installation does not work with a tablet or phone.
 
-   - If you have an ESP32-S3-BOX-3, select the **Connect** button below to show the current ports. Do not connect the ESP32-S3-BOX-3 yet.
-   - If your device does not readily connect for flashing, you may need to put your ESP32-S3-BOX-3 into "flash mode" by holding the "boot" button (left side upper button), tapping the "reset" button (left side lower button), and then releasing them both. Wait a few seconds before attempting to connect.
-   - If your browser does not support web serial, you will see a warning instead of a button.
+   - If you have an ESP32-S3-BOX-3, select the **Connect** button below to display a list of available USB devices. Do not connect the ESP32-S3-BOX-3 yet.
+   - If your ESP32-S3-BOX-3 does not appear in the list of devices presented by your browser:
+     - You can manually invoke "flash mode" by holding the "boot" button (left side upper button) as you tap the "reset" button (left side lower button); after a few seconds, the ESP32-S3-BOX-3 should appear in the list of USB devices presented by your browser.
+     - **Note:** this approach will require you to manually reset the ESP32-S3-BOX-3 (by again using the reset button) after the initial installation completes. You'll also need to reconnect to it (using the Connect button as before) to configure Wi-Fi.
```

Comment:
```suggestion
     - Follow the steps until step 3. After selecting the **Next** button, on the S3-Box, select the "Reset" button again.
     - Then, select the blue **Connect button** again, select the USB device and follow the prompts to configure the Wi-Fi.
```

Reply:
Instead of `select the "Reset" button again` it should be `tap the reset button (on the S3-Box) again` -- this is a physical button so I don't think "select" is correct. 😇

Reply:
good point, thank you. fixed.